Assets/GothicVR/Scripts/Debugging/FeatureFlags.cs

```diff
@@ -27,6 +27,8 @@ public enum SunMovementPerformance
         public bool CreateWaypoints;
         [Tooltip("Leave blank if you want to spawn normal.")]
         public string spawnAtSpecificFreePoint;
+        [Tooltip("True will render all pickables with dynamic attach points")]
+        public bool dynamicGrab;
```

Comment:
Let's call it something like vobItemsDynamicAttach as it is more clear what it is for.